SimPPS/DirectSimProducer/python/simPPS2021_cfi.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
The same here - better not adding 2021.

Reply:
@civanch Feasible indeed.

Reply:
@forthommel , please substitute 2021 to 2022 - in all other places it is implemented.

Reply:
@civanch please see https://github.com/cms-sw/cmssw/pull/38280#discussion_r931842961